dao/src/main/scala/io/codegalaxy/domain/dao/TopicDao.scala

```diff
@@ -30,15 +52,38 @@ class TopicDao(val ctx: CodeGalaxyDBContext) extends CommonDao
   }
 
   private def listQuery(): IO[Seq[Topic], Effect.Read] = {
-    for {
-      topics <- ctx.run(topics.leftJoin(topicsStats).on(_.id == _.id)
-        .sortBy { case (r, _) => r.id }
-      )
-    } yield {
-      topics.map { case (entity, maybeStats) =>
-        Topic(entity, maybeStats)
-      }
-    }
+    ctx.runQuery(
+      sql =
+        """SELECT
+          |  x3.id            AS _0,
```

Comment:
Just curious, is it required to have the part ` AS _0`?

Reply:
Yes, its **required**, but only if joining tables **with the same field names** (like `id`, in this case).

For some reason the JavaScript wrapper lib `web-sql` that we use for SQLite do not respect the fields scoping, so fields with the same names are clashing in the result objects.